playbooks/roles/network_info/tasks/main.yaml

```diff
@@ -0,0 +1,30 @@
+---
+- name: Read network info if present
+  slurp:
+    src: &networkinfo /etc/dev-install-network-info.yaml
```

Comment:
why do you use anchors?

Reply:
Only to DRY where it's not worth, for eg, setting an ansible fact. I don't use them very consistently, though.